index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+
+
+    <title>Jean Michel Basquiat </title>
+    
+    <!--
+    <style type="text/css">
+      body {
+        font-size: 20px;
+      }
+      p.main {
+        color: black;
+      }
+      .main {
+        color: black;
+        font-weight: bold;
+
+      } 
+      -->
+      
+    </style>
+  </head>
+  <body>
+
+    <h1>Jean Michel Basquiat</h1>
+    
+    <nav>
+      <a href="index.html">About</a>
+      <a href="experience.html">Experience</a>
+    </nav>
+
+    <hr>
+
+    <img src="jean.png">
```

Comment:
@elysemclark You just need to change the case of the file name. Your file is named **J**ean.png but you have jean.png. Other than that, we're good to go!